cmake/checks/check_01_cxx_features.cmake

```diff
@@ -463,3 +468,64 @@ CHECK_CXX_SOURCE_COMPILES(
   "
   DEAL_II_HAVE_ISFINITE)
 
+
+#
+# Check that we can use feenableexcept through the C++11 header file cfenv:
```

Comment:
Ah, that was of course a rookie mistake, mismatching test and use. Anyway, I'm ok with the patch if @tjhei is.